src/main/java/com/barterbay/app/domain/dto/user/UserDTO.java

```diff
@@ -1,23 +1,25 @@
-package com.barterbay.app.domain.dto;
+package com.barterbay.app.domain.dto.user;
 
+import com.barterbay.app.domain.dto.good.GoodCreatedDTO;
 import com.barterbay.app.validation.annotation.UserUniqueEmail;
 import lombok.Getter;
 import lombok.Setter;
 
 import javax.validation.constraints.Email;
 import javax.validation.constraints.NotBlank;
+import java.util.Set;
 
 @Getter
 @Setter
 public class UserDTO {
-
   @NotBlank(message = "Email cannot be empty")
   @Email(message = "Please enter a valid email address")
   @UserUniqueEmail
   private String email;
-
-  @NotBlank(message = "First name can not be empty")
+  @NotBlank(message = "First name cannot be empty")
   private String firstName;
-
-  // TODO complete this DTO
+  private Long id;
+  private String lastName;
+  private String password;
+  private Set<GoodCreatedDTO> goods;
```

Comment:
`private Set<GoodCreatedDTO> goods;` - why did you add it here? Do you use this UserDTO somewhere and do you need goodDTO inside?